test/Datadog.Trace.ClrProfiler.IntegrationTests/Helpers/SerializableDictionary.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public class SerializableDictionary : IXunitSerializable, IEnumerable<KeyValuePair<string, string>>
```

Comment:
I'm surprised that xUnit does not support serializing ` Dictionary<string, string>` out of the box. 🤔

Reply:
Yeah, I need to double check, as I initially assumed the dictionary was the problem, but later found that `HttpStatusCode` was an issue. But given even `Guid` is not serializable by default, I strongly suspect the dictionary is not either 😉

Reply:
Yep, confirmed, you can't use `Dictionary<string, string>` 🙄

Reply:
Thanks for checking!